pom.xml

```
@@ -23,7 +23,7 @@
   <repositories>
     <repository>
       <id>codehaus</id>
-      <url>http://http://repository.codehaus.org/</url>
+      <url>https://http://repository.codehaus.org/</url>
```

NIce catch but should this be:
```suggestion
      <url>https://repository.codehaus.org/</url>
```